_coffee/application.coffee

```diff
@@ -134,6 +139,15 @@ categories =
 53143469 # ios-demos
 54135763 # accessibilitools
 50032733 # novoda.github.io
+2168934  # public-mvn-repo
```

Comment:
It is automatically added to the Others section. Should I add it to the `exclude` sections instead?